content/integrations/chef.md

```diff
@@ -5,23 +5,73 @@ kind: integration
 
-* Get real-time reports on Chef client runs
-* Track key Chef performance metrics across all your servers
-* Quickly identify and discuss failed Chef runs with your team
+We also provide a Chef [Execution and Report Handler](https://docs.chef.io/handlers.html) that can capture `chef-client` failures as well as metrics related to the Chef run, such as timing and resources updated.
 
 ## Setup
-### Installation
+### Deploying the Agent
+
+The [Datadog Chef cookbook](https://supermarket.chef.io/cookbooks/datadog) is available to automate your Datadog Agent installation and configuration.
+
+Install the latest released version of the Datadog Chef cookbook from the [Supermarket](https://supermarket.chef.io/cookbooks/datadog) via knife, and upload to your Chef Server:
+
+    knife cookbook site install datadog
+    knife cookbook upload datadog
+
+And follow the instructions for your tool to upload the cookbook to your Chef Server.
+
+Before adding the cookbook's recipe to your node's `run_list`, you must add your Datadog account credentials such as API keys via Chef attributes.
+
+This is commonly done via `role` or `environment` files, or another cookbook declaring the attributes.
+
+Here is an example of a `base.rb` role file, typically applied to every host in an organization.
+
+{{< snippet-code-block file="guides-chef-base-role-agent.rb" >}}
+
+Note that there are two keys needed. Your API Key can be found in Datadog, under the Integrations => API menu item, or click [this link](https://app.datadoghq.com/account/settings#api) to log in and go there directly.
+
+Then, on the same page, you must create an Application Key for use with Chef. You may name the key whatever you wish, we recommend something like 'chef_appkey' or something of that nature.
+
+Provide both values in the attributes as shown above.
+
+Then upload your role file to Chef Server like so:
+
+    knife role from file roles/base.rb
+
+The next time Chef runs, it should install the Agent and set the configuration file with the API and application keys.
+
+**NOTE:** If you are using another cookbook to define these attributes, use a higher attribute precedence level than `default`.
+
+### Report Handler
+
+Datadog offers a Chef Report Handler which reports metrics and events from your Chef runs to Datadog. Once installed the report handler will submit metrics on Chef run timing and resource changes. Events are also created to allow tracking of Chef run success and failure rates.
+
+This has the added value of bringing the output of a Chef run back to Datadog's Event stream, so failures can be highlighted quickly, discussed amongst the team, and resolved.
+
+Successes typically will be found in the "Low" priority, whereas failures are of "Normal" priority, and when the same node passes the Chef run, then it is put pack into "Low" priority.
```

Comment:
Success will be found in the "Low" priority, whereas failures are of "Normal" priority, and when the same node passes the Chef run, then it is put pack into "Low" priority.

Reply:
👍